Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
   "crypto",                       # Cryptographic primitives and their interfaces.
   "consensus",                    # Consensus related logic.
   "chainstate",                   # Code on chainstate of blocks and transactions.
+  "chainstate/launcher",          # Tools to set up chainstate and storage together.
```

Comment:
Btw, this is not necessary. Launcher is a dependency of node, so it'll be included automatically.

My understanding is that we should include things here that we want to be autonomously built. There are many things that should be removed from here anyway, like crypto. Maybe I'll create another PR and do that.